stl/inc/bitset

```diff
@@ -347,11 +346,11 @@ public:
     _NODISCARD _CONSTEXPR23 basic_string<_Elem, _Tr, _Alloc> to_string(
         const _Elem _Elem0 = static_cast<_Elem>('0'), const _Elem _Elem1 = static_cast<_Elem>('1')) const {
         // convert bitset to string
-        basic_string<_Elem, _Tr, _Alloc> _Str;
-        _Str.reserve(_Bits);
-
-        for (auto _Pos = _Bits; 0 < _Pos;) {
-            _Str.push_back(_Subscript(--_Pos) ? _Elem1 : _Elem0);
+        basic_string<_Elem, _Tr, _Alloc> _Str(_Bits, _Elem0);

```

Comment:
I think that's a good case for `string::resize_and_overwrite`.
We can make internal `string::_Resize_and_overwrite` independent from standard mode, if such does not exist yet.

Reply:
This can also be done in a followup PR, I think.

Reply:
Filed #3858 for followup.